### ISSUE TITLE
AppleALC Support

### ISSUE THREAD

**Issue by khronokernel** [dortania/OpenCore-Legacy-Patcher#76]
Currently in the patcher we used a half-broken VoodooHDA to try and achieve audio on pre-2011 machines(excluding MacPro4,1 and 5,1). We've created a new branch to add support via AppleALC however needs testing:

* [AppleALC Support PR #75](https://github.com/dortania/Opencore-Legacy-Patcher/pull/75)

For users who wish to help, please run the above patcher on an unmodified Big Sur install and upload an [Audio Dump](https://github.com/dortania/Opencore-Legacy-Patcher/blob/applealc/Dump-Audio.command)(Folder will be created on your desktop). This should help us find the issue within AppleALC and hopefully have full working audio

**Comment by khronokernel** [comment]
With version 0.0.19, we've added on-disk patching to OCLP. This means that we've brought over AppleHDA patching as a way to satisfy users currently. Note that with on-disk patching, the following are limited:

- No Delta updates
  - OTA updates are still supported, however they'll be ~12GB for each OS update
- No FileVault support
- No SIP

To run these patches, ensure you've built and booted OpenCore with SIP and SecureBootModel disabled in patcher settings. Otherwise patcher will refuse to start.

Note: we do still plan to add AppleALC support, this is only a temporary work-around until we've resolved HDA patching
